packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -195,6 +196,7 @@ export class ObjectSetListenerWebsocket {
     const sub: Subscription<Q, P> = {
       listener: fillOutListener<Q, P>(listener),
       objectSet,
+      primaryKeyPropertyName: objDef.primaryKeyApiName,
```

Comment:
a few lines up we load the `objDef` right? I dont think we need to pass this in?

Reply:
We pass this in to use in #handleMessage_objectSetChanged, where we use the API name to delete. In #handleMessage_objectSetChanged, we dont load the def